app/src/main/java/com/example/bikini_android/repository/feed/FeedRepositoryImpl.kt

```diff
@@ -62,6 +66,24 @@ class FeedRepositoryImpl private constructor() : FeedRepository {
             }
     }
 
+    override fun addFeedToRemote(
+        feed: Feed,
+        imageFiles: List<MultipartBody.Part>
+    ): Single<DefaultResponse> {
+        return ApiClientHelper
+            .createMainApiByService(ImagesService::class)
+            .uploadImages(imageFiles)
+            .flatMap {
+                it.result?.url?.let {
+                    ApiClientHelper.createMainApiByService(FeedService::class)
+                        .addFeed(feed.apply {
+                            imageUrl = it
+                        })
+                }
+            }
```

Comment:
override fun addFeedToRemote(
        feed: Feed,
        imageFiles: List<MultipartBody.Part>
    ): Single<ImagesResponse> {
        val response = ApiClientHelper.createMainApiByService(ImagesService::class)
            .uploadImages(imageFiles)
            .subscribeOn(Schedulers.io())
            .subscribe { t1, t2 ->
                Log.e("Jaewon", "DONE")
            }

        return ApiClientHelper
            .createMainApiByService(ImagesService::class)
            .uploadImages(imageFiles)
            // .flatMap {
            //     val list = it.result
            //     with (list) {
            //
            //     }
            //     it.result?.url?.let {
            //         ApiClientHelper.createMainApiByService(FeedService::class)
            //             .addFeed(feed.apply {
            //                 imageUrl = it
            //             })
            //     }
            // }
            .subscribeOn(Schedulers.io())
    }